x-pack/test/functional/apps/infra/home_page.ts

```diff
@@ -118,6 +126,92 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
         await pageObjects.infraHome.getWaffleMap();
         // await pageObjects.infraHome.getWaffleMapTooltips(); see https://github.com/elastic/kibana/issues/137903
       });
+
+      describe('Asset Details flyout', () => {
```

Comment:
I know that we have tests on the host view here I tried to mostly cover the differences